src/libs/api-service.js

```diff
@@ -68,4 +68,12 @@ export default class ApiService {
         return response.data.objects;
       });
   }
+
+  subjectAccessRequest(email: string) {
+    return this.client
+      .post('/subject_access_request', { email: email })
+      .then(response => {
+        return response.data.objects;
+      });
```

Comment:
As you've done above, you could drop the curlies.
```suggestion
.then(resp => resp.data.objects);
```